src/compile/repeat.ts

```diff
@@ -40,7 +40,7 @@ export class RepeatModel extends BaseConcatModel {
     // cross product
     for (const rowField of row) {
       for (const columnField of column) {
-        const name = (rowField ? '_' + rowField : '') + (columnField ? '_' + columnField : '');
+        const name = (rowField ? `__row_${rowField}` : '') + (columnField ? `__column_${columnField}` : '');
```

Comment:
Please use the `internalField` method so it's easier to track where internal fields are being created.

Reply:
I'm not sure what you're talking about.  This is setting a spec "name".

Reply:
Ignore me.